javascript/javascript1/week3/notes.js

```diff
@@ -0,0 +1,44 @@
+const notes = [];
+
+function saveNote(content1, id1) {
+  notes.push({
+      content: content1,
+      id: id1
+  })
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes); 
+
+// getNote
+function getNote(id) {
+    for(let i = 0;i < notes.length; i++)
+    {
+        if(notes[i].id === id)
+        {
+            return notes[i];
+        }
+        else {
```

Comment:
You could be a little more specific here, saying if id is a string or undefined.

Reply:
Done